pkg/agent/run.go

```diff
@@ -346,11 +345,7 @@ func gatherData(config CombinedConfig, dataGatherers map[string]datagatherer.Dat
 			continue
 		}
 
-		if count >= 0 {
-			logs.Log.Printf("successfully gathered %d items from %q datagatherer", count, k)
-		} else {
-			logs.Log.Printf("successfully gathered data from %q datagatherer", k)
-		}
+		log.Info("Successfully gathered", "count", count, "name", k)
```

Comment:
The fetch count was introduced in https://github.com/jetstack/jetstack-secure/pull/537. I decided to always include the count in the structured logs, to simplify the code a bit and so that it's clear to the user when 0 items are found.
 * https://github.com/jetstack/jetstack-secure/pull/537

Reply:
Done.